code/modules/cm_marines/dropship_equipment.dm

```diff
@@ -136,85 +136,85 @@
 	var/screen_mode = 0 //used by the dropship console code when this equipment is selected
 	var/point_cost = 0 //how many points it costs to build this with the fabricator, set to 0 if unbuildable.
 
-	Dispose()
-		if(ammo_equipped)
-			cdel(ammo_equipped)
-			ammo_equipped = null
-		if(linked_shuttle)
-			linked_shuttle.equipments -= src
-			linked_shuttle = null
-		if(ship_base)
-			ship_base.installed_equipment = null
-			ship_base = null
-		if(linked_console)
-			if(linked_console.selected_equipment && linked_console.selected_equipment == src)
-				linked_console.selected_equipment = null
-			linked_console = null
-		. = ..()
-
-	attackby(obj/item/I, mob/user)
-
-		if(istype(I, /obj/item/powerloader_clamp))
-			var/obj/item/powerloader_clamp/PC = I
-			if(PC.loaded)
-				if(ship_base && uses_ammo && !ammo_equipped && istype(PC.loaded, /obj/structure/ship_ammo))
-					var/obj/structure/ship_ammo/SA = PC.loaded
-					if(SA.equipment_type == type)
-						playsound(src, 'sound/machines/hydraulics_1.ogg', 40, 1)
-						if(do_after(user, 30, FALSE, 5, BUSY_ICON_BUILD))
-							if(!ammo_equipped && PC.loaded == SA && PC.linked_powerloader && PC.linked_powerloader.buckled_mob == user)
-								SA.forceMove(src)
-								PC.loaded = null
-								playsound(src, 'sound/machines/hydraulics_2.ogg', 40, 1)
-								PC.update_icon()
-								to_chat(user, "<span class='notice'>You load [SA] into [src].</span>")
-								ammo_equipped = SA
-								update_equipment()
-					else
-						to_chat(user, "<span class='warning'>[SA] doesn't fit in [src].</span>")
-
-			else if(uses_ammo && ammo_equipped)
-				playsound(src, 'sound/machines/hydraulics_2.ogg', 40, 1)
-				if(do_after(user, 30, FALSE, 5, BUSY_ICON_BUILD))
-					if(ammo_equipped && PC.linked_powerloader && PC.linked_powerloader.buckled_mob == user)
-						playsound(src, 'sound/machines/hydraulics_1.ogg', 40, 1)
-						if(!ammo_equipped.ammo_count)
-							ammo_equipped.loc = null
-							to_chat(user, "<span class='notice'>You discarded the empty [ammo_equipped.name] in [src].</span>")
-							cdel(ammo_equipped)
-						else
-							ammo_equipped.forceMove(PC.linked_powerloader)
-							PC.loaded = ammo_equipped
+/obj/structure/dropship_equipment/Dispose()
+	if(ammo_equipped)
+		cdel(ammo_equipped)
+		ammo_equipped = null
+	if(linked_shuttle)
+		linked_shuttle.equipments -= src
+		linked_shuttle = null
+	if(ship_base)
+		ship_base.installed_equipment = null
+		ship_base = null
+	if(linked_console)
+		if(linked_console.selected_equipment && linked_console.selected_equipment == src)
+			linked_console.selected_equipment = null
+		linked_console = null
+	. = ..()
+
+/obj/structure/dropship_equipment/attackby(obj/item/I, mob/user)
+
+	if(istype(I, /obj/item/powerloader_clamp))
+		var/obj/item/powerloader_clamp/PC = I
+		if(PC.loaded)
```

Comment:
Christ this code in this proc is horrific, could you try to flatten the indentation a bit